packages/customize-widgets/src/controls/sidebar-section.js

```diff
@@ -71,10 +75,20 @@ export default function getSidebarSection() {
 						.closest( '.wp-full-overlay' )
 						.addClass( 'section-open' );
 					this.contentContainer.removeClass( 'open' );
-					this.contentContainer.one( 'transitionend', () => {
-						this.contentContainer.removeClass( 'busy' );
-						args.completeCallback();
-					} );
+				}
+
+				const handleTransitionEnd = () => {
+					this.contentContainer.removeClass( 'busy' );
+					args.completeCallback();
+				};
+
+				if ( isReducedMotion ) {
+					handleTransitionEnd();
```

Comment:
This is to fix a bug in one of the existing tests which surfaces itself during the migration. For some reason, `transitionend` event below never fires in CI, but I couldn't reproduce it locally. Anyway, it makes sense to skip the event when the user agent prefers reduced motion.

Reply:
Yep! In fact, both these two tests (`customizing-widgets` and `new-post`) are included here only for demonstration purposes, and they could be separated into new PRs along with the fix here. This should help minimize the changes introduced in this PR and make it easier to review.